test/unit/commands/scan.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {expect, test} from '@oclif/test';
+
+describe('Scan Unit Tests', function () {
+  this.slow(3000);
+  this.timeout(5000);
+  test
+    .stdout()
+    .command(['scan', '--dummy', '--file', '../test/rules.json', '--verbose'])
```

Comment:
Same notes as the other files re: assertions

But my main note here is that we don't really need test coverage on throwaway code that should be deleted & never invoked; we'll now need to delete both the dummy class and tests

Reply:
Ok, that makes sense. I was trying to keep the code coverage up, but I won't do that moving forward for throwaway code